#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("eps_angle", double_t, 0, "eps angle", 0.2, 0.0, pi)
gen.add("angle_offset", double_t, 0, "angle_offset", 0.0, 0.0, pi / 2.0)
exit (gen.generate (PACKAGE, "jsk_pcl_ros", "NormalDirectionFilter"))
